__tests__/components/Header.test.js

```diff
@@ -1,45 +1,28 @@
-// Copyright 2018 Stanford University see LICENSE for license
```

Comment:
This file moved to `__tests__/components/home/Header.test.js` and the file that was in `__tests__/components/editor/Header.test.js` has replaced it.